### PR TITLE
Lana tests

### PR DESCRIPTION
**Pull request by lanachiad**

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -1,5 +1,19 @@
 <h1>Registration Form</h1>
 
+<% if @user.errors.any? %>
```
Comment (louisgaeta1) [inline app/views/users/new.html.erb]: lets put this in a partial

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -2,4 +2,86 @@
...
+    it "assigns a user to @user" do
+      get :login
+      expect(assigns(:user)).to be_a User
+    end
+  end
+
+  describe "POST#loggingin" do
+    context "when valid params are passed" do
+      it 'should be successful to log in' do
+        response.should be_success
```
Comment (louisgaeta1) [inline spec/controllers/users_controller_spec.rb]: should this be expect(response).to have_http_status(:success)?

File: spec/controllers/users_controller_spec.rb [hunk 3]
```diff
@@ -2,4 +2,86 @@
...
+      end
+
+      it 'should start a session when logged in'
+    end
+
+    context "when invalid params are passed" do
+      it 'should not be successful to log in' do
+      	post :create, params: { user: { username: "" } }
+        response.should_not be_success
```
Comment (louisgaeta1) [inline spec/controllers/users_controller_spec.rb]: same here

File: spec/controllers/users_controller_spec.rb (hunk 2 above)
Comment (alexnhill) [reply spec/controllers/users_controller_spec.rb]: Agreed with Lou- success code would be 200

File: spec/controllers/users_controller_spec.rb (hunk 3 above)
Comment (alexnhill) [reply spec/controllers/users_controller_spec.rb]: expect(response).to have_http_status(400 or 500?)

File: app/views/users/new.html.erb (hunk 1 above)
Comment (lanachiad) [reply app/views/users/new.html.erb]: Did it!

**Pull request merged**